docker/service/src/main/java/uk/nhs/adaptors/scr/models/GpSummary.java

```diff
@@ -3,7 +3,7 @@
 import lombok.Getter;
 import lombok.Setter;
 import lombok.extern.slf4j.Slf4j;
-import org.hl7.fhir.r4.model.Bundle;
```

Comment:
Can you leave it as explicit rather than using * - if you need to import another that's ok, but please don't use *.

Reply:
Fixed!